components/kyma-environment-broker/internal/storage/postsql/read.go

```diff
@@ -605,7 +605,7 @@ func (r readSession) GetOperationStatsForOrchestration(orchestrationID string) (
 
 func (r readSession) GetInstanceStats() ([]dbmodel.InstanceByGlobalAccountIDStatEntry, error) {
 	var rows []dbmodel.InstanceByGlobalAccountIDStatEntry
-	_, err := r.session.SelectBySql(fmt.Sprintf("select global_account_id, count(*) as total from %s group by global_account_id",
+	_, err := r.session.SelectBySql(fmt.Sprintf("select global_account_id, count(*) as total from %s where deleted_at = '0001-01-01T00:00:00.000Z' group by global_account_id",
```

Comment:
This expression is format dependable. This is risky. Is there no other way to query?

Reply:
This is the default value for deleted_at in the database. I couldn't think of a better solution, maybe you have an idea?

Reply:
Is there an index containing deleted_at?

Reply:
I don't see any declarations in the schema.

Reply:
It seems index is not necessary.